deploy/kabanero-customresources.yaml

```diff
@@ -254,6 +254,20 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ''
+  - image.openshift.io
+  resources:
+    - imagesignatures
+  verbs:
+    - create
+    - delete
+    - get
+- nonResourceURLs:
+  - '/staging/*'
```

Comment:
@toshiyamamoto can you explain the `/staging/` permission?

Reply:
When an external image registry with sample look aside signature server is used, after signing an image, a sign script puts a generated signature to a signature server by using curl, and k8s checks whether outgoing http request is allowed by checking the service account roles.  This specific uri is used for accessing the sample signature server. 
If the customer has their own signature server and it's url is different from the sample, then they need to update the role to match their environment.